### ISSUE TITLE
Unable To Play Stream

### ISSUE THREAD

**Issue by Azoto1985** [AirenSoft/OvenMediaEngine#13]
I'm able to connect to the server with OBS , the stream is ingested as well.
When i'm trying to connect with the OvenPlayer i see only a black screen with this error from the console
![error](https://user-images.githubusercontent.com/46760800/51274097-911c0200-19ce-11e9-8d03-5e2088b41e98.jpg)
How to fix? 
I've also tried configuring the SSL certificate in order to play the wss:// but i still have those errors.

Sometimes i've also get this error

[01-16 20:38:23.188] W 18458 TranscodeStream | transcode_stream.cpp:430  | Queue(filtered) is full, Please check your system

**Comment by Azoto1985** [comment]
I've configured the OBS with baseline H264 ultrafast but still black screen.

**Comment by sfreet** [comment]
Hello Azoto.

The attached log does not seem to be a problem.
Yellow messages are warnings or information, not errors. (A red message means an error)
"Queue(filtered) is full..." message is also not a problem unless it appears very often.

I need more information.
1. OvenPlayer configuration (modified JavaScript code)
2. Server.xml
3. WebBrowser - name, version

If possible, check again your setting based on [dimiden](https://github.com/dimiden)'s last comment in issue #10

**Comment by Azoto1985** [comment]
Dear sfreet, thank you for your support. Infact everything seems ok. I discovered the problem..
The OvenMedia accept only the phisical IP of the Server instead of the Failover IP.

Now it's working but i still have those warnings :+1: 

[01-17 18:53:42.499] W 6381 TranscodeStream | transcode_stream.cpp:430  | Queue(filtered) is full, Please check your system

It seemd that the transcoder got some problem and also the quality of the Video Output in the OvenPlayer is bad.
Is there some configuration to avoid this?

Thanks for your help

**Comment by sfreet** [comment]
Hello Azoto.
I'm glad to hear that some of the problems have been resolved.

1. Video quality problem

You probably set the encoding resolution in Server.xml.
I assume it is (480*320). -> Based on the debug message you attached

Try changing the encoding resolution in Server.xml after comparing it with the OBS resolution settings.

First, check the resolution setting in OBS studio.
It may be better to change it to a different value in Server.xml (for example, 800*600 or 1024*768).
However, that setting causes a little more CPU load, and consequently A message can be output more often.
In this case, you need to determine the settings according to the specifications of the hardware.


2. Debug message problem

It can occurs when the encoding speed is slower than the input speed of the stream. (but, that's not all)

You can reduce the encoding load by removing one of the VP8 or H264 options in Server.xml, as follows:
```
<Profile>FHD_H264</Profile>
<!--<Profile>FHD_VP8</Profile>-->
```
But it can be a problem depending on the web browser you use.
If you're using the latest version of Chrome or Safari, you'll have no problem.

If you tell me more information (eg CPU, memory, OBS settings, Server.xml ...), I may be able to give you more useful information.

PS: I am currently fixing the debug messages, so you can safely ignore the "Queue is full ..." message unless it happens very frequently.

**Comment by Azoto1985** [comment]
Dear sfreet, thank you again.
I've checked the Server.xml configuration and i've set the application to 1280*720 , i've also set the Bitrate to the original value : 1000000 . 
Now evything seems ok.
I've discovered a bug when you are broadcasting with OBS and you lost the connection, or change wifi connection while are you streaming , when you trying to reconnect to the same AppName you get this error from the console (see screenshot).
If you try to connect to a different AppName no problem. Is there a way to auto-disconnect source from the configuration? 
It seems that actually the only way to re-connect the same AppName is to restart totally the OME and kill the previous OME process.
Regards
![screenshot_20190118-133312](https://user-images.githubusercontent.com/46760800/51387484-6e9cfc80-1b26-11e9-9d81-eaa69f3d20a8.jpg)

**Comment by Azoto1985** [comment]
Dear sfreet, thank you for your help, i've experienced this issue only when changing the connection while streaming, it seems that the app_name created freeze and you need to restart the OvenMediaEngine. 
Regards

**Comment by bongjj77** [comment]
Hi Azoto
Fixed a input stream(rtmp) network shutdown bug in the last updated version. Check it

The network card connection termination problem should wait about 10 seconds.
or 
If you enter  in the server.xml file 
<Providers>
<RTMP>
**<OverlapStreamProcess>refresh</OverlapStreamProcess>**
</RTMP>
</Providers>
, It will be possible to reconnect immediately

**Comment by sfreet** [comment]
This issue was resolved by @bongjj77 in version 959d8e1f6a6362ec16ffdae5986014a9aae116a9. 
I close this issue. If the issue is not resolved, please let us know again.
Thanks again for your contributions Azoto~
